ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4521,124 +4521,124 @@
-		<entry key="FrontEndMenu_SetName_ELLY" translation="Miss Elly's Round Up"/>
+		<entry key="FanfareCareerSetUnlocked_ELLY" translation="Miss Elly sta cercando un aiuto per il ranch, ma non è il bestiame che sta radunando, sono i visitatori!"/>
+		<entry key="FanfarePackCompletion_ELLY_1" translation="Hai collezionato tutte le stelle di Bronzo per 'Il raduno di Miss Elly'."/>
+		<entry key="FanfarePackCompletion_ELLY_2" translation="Hai collezionato tutte le stelle d'Argento per 'Il raduno di Miss Elly'."/>
+		<entry key="FanfarePackCompletion_ELLY_3" translation="Hai collezionato tutte le stelle d'Oro per 'Il raduno di Miss Elly'."/>
+		<entry key="FanfarePackCompletion_ELLY_All" translation="Hai completato ogni obiettivo per 'll raduno di Miss Elly'!"/>
+		<entry key="FrontEndMenu_ScenarioDetails_DINE" translation="Il ristorante di Miss Elly ha delle strutture eccezionali, compreso un grande coaster di legno per un pò di eccitazione! Ma la posizione remota signica non avere molti visitatori e il suo potenziale non viene realizzato. Riesci a costruire in questo posto e attirarci le folle?"/>
+		<entry key="FrontEndMenu_ScenarioDetails_MARS" translation="Hanno girato alcuni film abbastanza famosi qua fuori nel deserto. Assomiglia a Marte, vero? Il posto non è più richiesto – è tutta computer grafica di questi tempi – ma la compagnia non vuole lasciarlo andare. Vogliono invece che tu lo trasformi in un parco a tema!"/>
+		<entry key="FrontEndMenu_ScenarioDetails_MINE" translation="Un poveraccio ha comprato questa miniera da un cercatore e ha chiesto alcuni grandi prestiti per pagarla. E' venuto fuori che è una truffa. La canaglia aveva salato la miniera! Ora sta cercando di trasformarla in un'attrazione turistica con alcuni fuochi d'artificio e con strade stile Vecchio West, ma non sta andando così bene come sperava e ha bisogno del tuo aiuto."/>
+		<entry key="FrontEndMenu_ScenarioName_DINE" translation="Il ristorante di Miss Elly"/>
+		<entry key="FrontEndMenu_ScenarioName_MARS" translation="Hangar della Nave Spaziale"/>
+		<entry key="FrontEndMenu_ScenarioName_MINE" translation="Torre della Miniera d'Oro"/>
+		<entry key="FrontEndMenu_SetDescription_ELLY" translation="Devi fare un bel colpaccio per riuscire qui. Puoi domare la frontiera selvaggia?"/>
+		<entry key="FrontEndMenu_SetName_ELLY" translation="Il raduno di Miss Elly"/>
 		<!--C3_Coasters-->
 		<entry key="Coaster_Bolt" translation="Bolt"/>
-		<entry key="Coaster_Bolt_Desc" translation="Shuttle roller coaster, allowing the train to travel forwards to an end point before then travelling along the track in reverse back to the station."/>
+		<entry key="Coaster_Bolt_Desc" translation="Montagna russa Shuttle, che permette  al treno di viaggiare in avanti fino ad un punto terminale prima di viaggiare all'indietro lungo il percorso fino alla stazione di partenza."/>
```

Comment:
Ci sono due spazi in "che permette  al", questo posso correggerlo anche io se vuoi, altrimenti segnatelo per la prossima pull

Reply:
Ok